boards/jiminy-mega256rfr2/doc.txt

```diff
@@ -133,4 +133,23 @@ This should take care of everything!
 
 RIOT's Makefile are configured to flash the jiminy using AVRDUDE. The bootloader automatically matches
 to the configured baud rate which is set for AVRDUDE. Rates of up to 500kBaud can be used.
- */
\ No newline at end of file
+
+# Special Notes
+
+Setting the TPS6274x output voltage to 1.8V leads to a undervoltage and thus triggers the brown-out reset.
+Disabling the brown-ouit detection with the fuses is recommended when a 1.8V supply is used.
+
+Reading out the fuses can be done with 
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U lfuse:r:-:i -v`
+
+The last line should read 
+`avrdude: safemode: Fuses OK (E:FE, H:D0, L:E2)`
+
+extended fuse = FE
+High fuse     = D0
+Low fuse      = E2
+
+To program this fuse setting execute following line 
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U lfuse:w:0xe2:m -U hfuse:w:0xd0:m -U efuse:w:0xfe:m`
```

Comment:
Wouldn't `0xfe` set the `BODLEVEL` bits to `0b110` which means 1,8V brown-out detection? I expected `0xff` (See Table 35-23 in the [reference manual](http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-8393-MCU_Wireless-ATmega256RFR2-ATmega128RFR2-ATmega64RFR2_Datasheet.pdf)).

Reply:
could you use the `-v ` option and post the whole output! 

Seems to be a usb atmega16 communication issue.

Reply:
The bootloader does not allow to flash fusebits. You need to usw an ISP programmer for that.